v1/root/root.go

```diff
@@ -55,11 +55,26 @@ func (rc *RootCommand) AddCommand(cmd *command.Command) {
 // Returns:
 //   - An error, if an error occurs during execution.
 func (rc *RootCommand) Execute() error {
+	// Required flags check
+	setFlags := make(map[string]bool)
+	var necessaryFlags []string
```

Comment:
The only thing I would change is the name of this local var, because in other places we use `required` and here is `necessary`